pycscope/__init__.py

```diff
@@ -598,7 +598,10 @@ def markTestlist(ctx, cst):
 if sys.hexversion < 0x03000000:
     tse = symbol.testlist
     test_or_star_expr = (symbol.test,)
-    testlist_comp = (symbol.testlist_comp, symbol.listmaker)
+    if sys.hexversion < 0x02070000:
```

Comment:
When I downloaded the 2.6.9 release source code I did not see testlist_comp, only testlist_gexp.  So can you find out which version of 2.6 it changed in and then use that?

Sorry it has take me so long to get back to this.